pkg/runctl/run_manager_test.go

```diff
@@ -1727,6 +1720,60 @@ func Test_RunManager_Log_Elasticsearch(t *testing.T) {
 		})
 	}
 
+	/**
+	 * Test: Various JSON values for spec.logging.elasticsearch.runID
+	 * are correctly passed as Tekton TaskRun input parameter.
+	 */
+	test = "SuppressRunID"
+	for _, tc := range []struct {
+		name               string
+		runIDJSON          string
+		expectedParamValue string
+	}{
+		{"none", ``, `null`},
+		{"null", `"runID": null`, `null`},
+		{"dummy", `"___dummy___": 1`, `null`},
+	} {
+		t.Run(test+"_"+tc.name, func(t *testing.T) {
+			// setup
+			pipelineRunJSON := fmt.Sprintf(fixIndent(`
+				{
+					"apiVersion": "steward.sap.com/v1alpha1",
+					"kind": "PipelineRun",
+					"metadata": {
+						"name": "dummy1",
+						"namespace": "namespace1"
+					},
+					"spec": {
+						"jenkinsFile": {
+							"repoUrl": "dummyRepoUrl",
+							"revision": "dummyRevision",
+							"relativePath": "dummyRelativePath"
+						},
+						"logging": {
+							"elasticsearch": {
+								%s
+							}
+						}
+					}
+				}`),
+				tc.runIDJSON,
+			)
+			t.Log("input:", pipelineRunJSON)
+			examinee, runCtx, cf := setupExaminee(t, pipelineRunJSON)
+
+			// exercise
+			resultError := examinee.createTektonTaskRun(runCtx)
+			assert.NilError(t, resultError)
+			// verify
+			taskRun := expectSingleTaskRun(t, cf, runCtx.pipelineRun)
+			param := findTaskRunParam(taskRun, TaskRunParamNameRunIDJSON)
+			assert.Assert(t, param != nil)
+			assert.Equal(t, tekton.ParamTypeString, param.Value.Type)
+			assert.Equal(t, tc.expectedParamValue, param.Value.StringVal)
+		})
+	}
+
```

Comment:
Can be covered by `Passthrough` test above, can't it?

Reply:
Right, It can be. I moved them up